WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustoms.swift

```diff
@@ -0,0 +1,69 @@
+import SwiftUI
+import WooFoundation
+
+struct WooShippingCustoms: View {
+    let informationIsCompleted: Bool
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    var body: some View {
+        HStack {
+            Text(Localization.customsTitle)
+                .font(.system(size: Layout.customsTitleFontSize, weight: .medium))
+                .foregroundColor(.primary)
+
+            Spacer()
+
+            Text(informationIsCompleted ? Localization.completedStatus : Localization.missingInfoStatus)
+                .font(.system(size: Layout.statusBadgeFontSize, weight: .medium))
```

Comment:
Same as above with the font size preventing dynamic font resizing. In this case I'd expect `.captionStyle()` should work here.

Reply:
Yeah, that makes sense. Done in https://github.com/woocommerce/woocommerce-ios/pull/14729/commits/8130522b485ded46f5c545185fec3ec253951c42